rules/integrations/endpoint/elastic_endpoint_security_behavioral.toml

```diff
@@ -0,0 +1,68 @@
+[metadata]
+creation_date = "2021/09/18"
+maturity = "production"
+updated_date = "2021/09/18"
+integration = "endpoint"
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Endpoint Security alert is received for Behavior Protection alerts.
+Enabling this rule allows you to immediately begin investigating your Endpoint alerts for Behavior Protection.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-endpoint.alerts-*"]
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Endpoint Security Behavioral Protection"
+risk_score = 47
+rule_id = "d516af98-19f3-45bb-b590-dd623535b746"
+rule_name_override = "rule.name"
+severity = "medium"
+tags = ["Elastic", "Endpoint Security"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.kind:alert and event.module:(endpoint and not endgame) and event.category: behavior
+'''
+
+
+[[rule.exceptions_list]]
+id = "endpoint_list"
+list_id = "endpoint_list"
+namespace_type = "agnostic"
+type = "endpoint"
+
+[[rule.risk_score_mapping]]
+field = "event.risk_score"
+operator = "equals"
+value = ""
```

Comment:
defined severity applies if missing I assume?

Reply:
same logic as the other rule, just copied